### PR TITLE
fix: rename idleTimeout option to keepAliveTimeout

### PR DESCRIPTION
**Pull request by ronag**
More consistent nameing.

### CODE REVIEW

**Review by mcollina — CHANGES_REQUESTED** [review]
Could you keep an alias to the previous option? Or throw if the previous option is passed in? Thanks.

**Comment by ronag** [comment]
@mcollina fixed

File: lib/core/client.js [hunk 1]
```diff
@@ -79,6 +79,10 @@ class Client extends EventEmitter {
   } = {}) {
     super()
 
+    if (idleTimeout !== undefined) {
+      throw new InvalidArgumentError('unsupported idleTimeout')
```
Comment (mcollina) [inline lib/core/client.js:83]:
```suggestion
      throw new InvalidArgumentError('unsupported idleTimeout, use keepAliveTimeout instead')
```

File: test/client-errors.js [hunk 2]
```diff
@@ -388,6 +388,13 @@ test('invalid options throws', (t) => {
     t.strictEqual(err.message, 'invalid keepAlive')
   }
 
+  try {
+    new Client(new URL('http://localhost:200'), { idleTimeout: 30e3 }) // eslint-disable-line
+  } catch (err) {
+    t.ok(err instanceof errors.InvalidArgumentError)
+    t.strictEqual(err.message, 'unsupported idleTimeout')
```
Comment (mcollina) [inline test/client-errors.js:395]:
```suggestion
    t.strictEqual(err.message, 'unsupported idleTimeout, use keepAliveTimeout instead')
```

**Review by mcollina — CHANGES_REQUESTED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**